src/contracts/transfer-gateway.ts

```diff
@@ -527,6 +536,47 @@ export class TransferGateway extends Contract {
       pending: response.getIsPending()
     }
   }
+
+  /**
+   * Retrieves the transfer gateway state
+   * @returns A promise that will be resolved the transfer gateway state.
+   * return null when state is undefined.
+   */
+  async getStateAsync(): Promise<ITransferGatewayState | null> {
```

Comment:
Instead of returning `null` just return a default state object with all fields set to zero, that way the calling code doesn't need to check for `null`.

Reply:
Fixed